packages/components/src/components/scroll-buttons/scroll-buttons.tsx

```diff
@@ -5,12 +5,26 @@ import { Component, h, JSX, Method, Prop, Watch } from '@stencil/core';
     styleUrl: 'scroll-buttons.scss',
     shadow: true
 })
+
 export class ScrollButtons {
     /**
      * Reference to the element with scroll on parent element.
+     *
      * @type {HTMLDivElement}
      */
     @Prop() scrollContainerElementRef: HTMLDivElement;
+
+    /**
+     * Updates scroll buttons enabled/disabled states when elementItems amount changes.
```

Comment:
This sounds like it happens automatically ("when elementItems amount changes"), but it is a handle to manually trigger recalculation of buttons state if needed. Please rephrase so it doesn't sound like stuff happens automatically.

Reply:
✅